### PR TITLE
sbt: add clean example

### PR DESCRIPTION
**Pull request by eshack94**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by eshack94** [comment]
What does `license/cla Expected — Waiting for status to be reported` refer to? Does this require action on my end to clear this unreported status check?

File: pages/common/sbt.md [hunk 1]
```diff
@@ -11,6 +11,10 @@
 
 `sbt new {{scala/hello-world.g8}}`
 
+- Delete all generated files (in the target directory):
```
Comment (blueskyson) [inline pages/common/sbt.md:14]: Do you mean working (current) directory?
```suggestion
- Delete all generated files (in the current directory):
```

Comment (eshack94) [reply pages/common/sbt.md:14]: @blueskyson No, I mean `target` directory, which is a directory that specific to scala builds. It is for cleaning previously compiled "stuff" prior to a new compile.

**Review by blueskyson — APPROVED** [review]

File: pages/common/sbt.md (hunk 1 above)
Comment (marchersimon) [reply pages/common/sbt.md:14]: I'd still remove the parentheses (I can't make a suggestion for some reason)

Comment (eshack94) [reply pages/common/sbt.md]: @marchersimon Made this change based on your suggestion. Ready for re-review :)

**Review by marchersimon — APPROVED** [review]
Thanks ＠eshack94
(btw, please don't force-push. We sqash all commits automatically)

**Comment by eshack94** [comment]
> Thanks ＠eshack94 (btw, please don't force-push. We sqash all commits automatically)

Sorry, I did not know that. I usually force push in my daily work to keep commit histories clean when amending a commit with no change to the commit message if the change is trivial. I will keep that in mind going forward.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**